security/compliance_operator/compliance-operator-release-notes.adoc

```diff
@@ -13,6 +13,52 @@ These release notes track the development of the Compliance Operator in the {pro
 
 For an overview of the Compliance Operator, see xref:../../security/compliance_operator/compliance-operator-understanding.adoc#understanding-compliance-operator[Understanding the Compliance Operator].
 
+[id="compliance-operator-release-notes-0-1-55"]
+== OpenShift Compliance Operator 0.1.55
+
+The following advisory is available for the OpenShift Compliance Operator 0.1.55:
+
+* link:https://access.redhat.com/errata/RHBA-2022:6657[RHBA-2022:6657 - OpenShift Compliance Operator bug fix update]
+
+[id="compliance-operator-0-1-55-new-features-and-enhancements"]
+=== New features and enhancements
+
+* `KubeletConfig` now checks the default configuration of the `KubeletConfig`. The configuration files are aggregated from all nodes into a single location. See xref:../../security/compliance_operator/compliance-operator-remediation.adoc#compliance-evaluate-kubeletconfig-rules[Evaluating `KubeletConfig` rules against default configuration values].
+
+* The `ScanSetting` Custom Resource now allows users to override the default CPU and memory limits of scanner pods through the `scanLimits` attribute.  For more information, see xref:../../security/compliance_operator/compliance-operator-troubleshooting.adoc#compliance-increasing-operator-limits[Increasing Compliance Operator resource limits].
+
+* A `PriorityClass` can now be set through `ScanSetting`. This will ensure the Compliance Operator is prioritized and minimizes the chance that the cluster will fall out of compliance. For more information, see xref:../../security/compliance_operator/compliance-operator-advanced.adoc#compliance-priorityclass_compliance-advanced[Setting `PriorityClass` for `ScanSetting` scans].
+
+[id="compliance-operator-0-1-55-bug-fixes"]
+=== Bug fixes
+
+* Previously, the Compliance Operator hard-coded notifications to the default namespace. If the Operator were installed in a non-default namespace, the notifications would not work as expected. Now, Notifications work in non-default namespaces. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2060726[*BZ#2060726*])
```

Comment:
The default namespace here refers to the openshift-compliance namespace. Because a namespace called "default" also exists on all OCP clusters, we might want to spell out openshift-compliance explicitly.